src/lib/journey/stages/recovery-codes.svelte

```diff
@@ -20,12 +20,12 @@
   export let formEl: HTMLFormElement | null = null;
   export let journey: StageJourneyObject;
   export let step: FRStep;
-  let codes: Array<string> = [];
 
+  let codes: Array<string> = [];
+  let name = 'New Security Key';
```

Comment:
Can we move these `let` statements down and group them with the existing `let` statements on lines 29 and 30?